apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -62,7 +130,7 @@ describe("Institution endpoints", () => {
         failOnStatusCode: false,
         method: "GET",
         headers: {
-          Authorization: `Bearer ${Cypress.env("USER_ACCESS_TOKEN")}`,
+          Authorization: `Bearer ${Cypress.env("NO_KEYS_USER_ACCESS_TOKEN")}`,
```

Comment:
Where is this coming from? Won't this token expire?

Reply:
It comes from this code: https://github.com/Universal-Connect-Project/ucp-hosted-app/pull/42/files/7235853217b8e9ac5a7a30f97027d350c456a456#diff-b52e65dd5f7d947e1df05c99feab0f6bc6a36f97d6f02312e3e706c99ea107cfR67-R79

Not hard-coded. This is code that Craig originally wrote, I just renamed the variable to be more specific about what it is.

Reply:
Ah I see now. We're good.